lib/influxdb/query/core.rb

```diff
@@ -14,9 +14,11 @@ def version
       # rubocop:disable Metrics/MethodLength
       def query(query, opts = {})
         denormalize = opts.fetch(:denormalize, config.denormalize)
+        json_streaming = !opts.fetch(:chunk_size, config.chunk_size).nil?
+
         params = query_params(query, opts)
         url = full_url("/query".freeze, params)
-        series = fetch_series(get(url, parse: true))
+        series = fetch_series(get(url, { parse: true }, json_streaming))
```

Comment:
Braces not needed, see https://github.com/influxdata/influxdb-ruby/pull/155/files#r71583399